pom.xml

```diff
@@ -7,7 +7,7 @@
     <!-- Arifact name and version information -->
     <groupId>net.snowflake</groupId>
     <artifactId>snowflake-ingest-sdk</artifactId>
-    <version>1.0.2-beta.7</version>
+    <version>1.0.2-beta.8</version>
```

Comment:
let's revert this in case there're other changes that we want to be included in beta.8, and we want this change to be the last change of the beta.8

Reply:
Revert of this would test failures. I suggest we keep it there, keep the PR open and only merge it just before `beta.8` release.